packages/discord.js/src/structures/MentionableSelectMenuInteraction.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const { Collection } = require('@discordjs/collection');
+const SelectMenuInteraction = require('./SelectMenuInteraction');
+/**
+ * Represents a {@link ComponentType.MentionableSelect} select menu interaction.
+ * @extends {SelectMenuInteraction}
+ */
+class MentionableSelectMenuInteraction extends SelectMenuInteraction {
+  constructor(client, data) {
+    super(client, data);
+    const { members, users, roles } = data.data.resolved || {};
+
+    if (members) {
+      /**
+       * Collection of the selected users
+       * @type {Collection<Snowflake, GuildMember|APIGuildMember>?}
+       */
+      this.members = new Collection();
```

Comment:
@discordjs/core should we have the collections set all times, and filled only when something is returned, or keep this as is? Personally I'd prefer if we always had the collections set